tests/phpunit/data/object-cache-annihilator.php

```diff
@@ -0,0 +1,523 @@
+<?php
+/**
+ * Object Cache Annihilator.
```

Comment:
I guess the source of this file is coming from https://github.com/polylang/object-cache-annihilator.
I suggest to maintain only the plugin and "require-dev" it with composer to re-use it.

Reply:
Done in e7895856639828f9502d63d492905ccf1594b797.